library/src/interactor_impl.cxx

```diff
@@ -266,15 +255,29 @@ class interactor_impl::internals
         else if (keySym == "Return")
         {
           ren->ResetCamera();
-          renWin->Render();
         }
         else if (keySym == "Space")
         {
           self->AnimationManager.ToggleAnimation();
-          renWin->Render();
         }
         break;
     }
+
+    if (checkColoring)
+    {
+      // TODO This is not super great, do better ?
+      bool useCell;
+      std::string arrayName;
+      int component;
+      renWithColor->GetColoring(useCell, arrayName, component);
+      self->Options.set("cells", useCell);
+      self->Options.set("scalars", arrayName);
+      self->Options.set("component", component);
```

Comment:
So I did not want to rewrite the Cycle logic in the renderer so here is how it works now:

1. Cycle in the renderer
2. Get the current coloring in the options
3. Later, SetColoring is called in UpdateDynamicOptions, but the options are the same, so nothing changes

Wdyt ?

Reply:
@Meakk